src/components/ConfigureEmail.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react'
+import { FormGroup, FormControl, Button } from 'react-bootstrap'
+import { SynapseClient, SynapseConstants } from 'synapse-react-client'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { UserBundle } from 'synapse-react-client/dist/utils/synapseTypes'
+import { getSearchParam, hexDecodeAndDeserialize } from 'URLUtils'
+import EditIcon from '../assets/RedEditPencil.svg'
+
+export type ConfigureEmailProps = {
+    returnToUrl: string
+}
+
+export const ConfigureEmail = (props: ConfigureEmailProps) => {
+    const { accessToken } = useSynapseContext()
+    const [ newEmail, setNewEmail ] = useState('')
+    const [ primaryEmail, setPrimaryEmail ] = useState('')
+    const [ userId, setUserId ] = useState('')
+    const [ emails, setEmails ] = useState<string[] | undefined>()
+    const [ editEmail, setEditEmail ] = useState(false)
+    
+    const btnStyle = {
```

Comment:
There is no current design for how the email management should look like. When there is a definite design I'll create a scss file for the styling.